src/docs/howto-transaction-process/change-transaction-process-in-ftw/index.md

```diff
@@ -1,73 +1,117 @@
 ---
-title: Change transaction process setup in FTW
+title: Change transaction process setup in Sharetribe Web Template
```

Comment:
"Change the transaction process setup" sounds a bit ambiguous, and the description says this article is about using a new transaction process. Could the title be e.g.
- Change your transaction process
- How to use a different transaction process
- Add a new transaction process (to the template)

Reply:
Fixed!